sample_apps/run_example.sh

```diff
@@ -744,6 +746,24 @@ function gen_policy_and_self_signed_cert() {
                    --operation=generate-policy-key                  \
                    --policy_key_output_file=policy_key_file.bin     \
                    --policy_cert_output_file=policy_cert_file.bin
+
+    elif [ "${SampleAppName}" = "multidomain_simple_app" ]; then
+        run_cmd "$CERT_UTILS"/cert_utility.exe                              \
```

Comment:
Code diffs from here onwards are basically cut-n-paste of commands from the `script` file, just reworked to fit the calling style of this script.